packages/console/src/console.ts

```
@@ -69,7 +71,17 @@ function logDeprecated(
   message: string,
   ...args: unknown[]
 ) {
-  if (!process.env.OMIT_INSTUI_DEPRECATION_WARNINGS) {
+  let shouldLogMessage = false
+
+  try {
+    shouldLogMessage = !process?.env?.OMIT_INSTUI_DEPRECATION_WARNINGS
+  } catch (e) {
+    if (e instanceof ReferenceError) {
```

What if other error occurs? Now we "swallow" it don't we? Is this what we want?